protected/humhub/modules/queue/LongRunningActiveJob.php

```diff
@@ -0,0 +1,13 @@
+<?php
+
+namespace humhub\modules\queue;
+
+use yii\queue\RetryableJobInterface;
+
+abstract class LongRunningActiveJob extends ActiveJob implements RetryableJobInterface
+{
+    public function getTtr()
+    {
+        return 60 * 60;
```

Comment:
@gevorgmansuryan Do you see a possibility to change this value? e.g. via config container definitions?

Reply:
done